pkg/epinio/pages/c/_cluster/applications/createapp/index.vue

```diff
@@ -100,18 +101,33 @@ export default Vue.extend<Data, any, any, any>({
       this.value.meta = this.value.meta || {};
       this.value.configuration = this.value.configuration || {};
       this.set(this.value.meta, changes.meta);
+      this.set(this.value.configuration, { settings: this.appChart.settings });
```

Comment:
Comment only - I don't think we often use this pattern (set does not overwrite the entire object, but does a partial apply per property)